opds_import.py

```diff
@@ -919,10 +906,17 @@ class OPDSImportMonitor(Monitor):
     """
     
     def __init__(self, _db, feed_url, default_data_source, import_class, 
-                 interval_seconds=3600, keep_timestamp=True,
-                 immediately_presentation_ready=False):
+                 interval_seconds=0, keep_timestamp=False,
+                 immediately_presentation_ready=False, force_reimport=False):
+
+        # We never keep a timestamp because we can track when the OPDS
+        # archive feed was last updated.
+        keep_timestamp = False
+        interval_seconds = 0
```

Comment:
These are already the default arguments, do we need to prevent subclasses from changing them?